.github/labeler.yml

```
@@ -11,6 +11,7 @@ i18n:
   - docs/src/content/docs/ja/**/*
   - docs/src/content/docs/zh-cn/**/*
   - docs/src/content/docs/pt-br/**/*
+  - docs/src/content/docs/pt/**/*
```

As Chris suggested on discord. Remember to move the files
```suggestion
  - docs/src/content/docs/pt-pt/**/*
```